src/os/ui/filter/im/filterimporter.js

```diff
@@ -91,6 +91,15 @@ os.ui.filter.im.FilterImporter.prototype.onParsingComplete = function(opt_event)
 };
 
 
+/**
+ * @param {os.filter.IFilterable} filterable
+ * @return {Array}
```

Comment:
The return type should match `getFilterColumns`, so `Array<os.ogc.FeatureTypeColumn>`.